swap/swap_test.go

```diff
@@ -134,22 +133,22 @@ func TestAllBalances(t *testing.T) {
 	swap, clean := newTestSwap(t, ownerKey)
 	defer clean()
 
-	if len(swap.balances) != 0 {
-		t.Fatalf("Expected balances to be empty, but are %v", swap.balances)
+	if balances, _ := swap.Balances(); len(balances) != 0 {
```

Comment:
We probably should not skip this error; although I am assuming that if there would be an error, then `balances` would be 0, thus the test would fail anyway (but the semantics of `Balances()` could change in the future and we would miss it here). Anyways, not picky on this one.

Reply:
done